### ISSUE TITLE
Needed - Schedule module

### ISSUE THREAD

**Issue by subdavis** [HackNC/fall2016#15]
This one will be a little tricker.  We need a dynamic schedule page.  It should be modular so that we could throw the html/css/js onto any page or drop it in any container and have it just work. Think of this less as a page and more as a module.

The schedule data will be delivered via JSON from some api service.  Don't worry about this yet. Imagine you'll be working with something like this: https://gist.github.com/suBDavis/536179a2f8673842355a

**Comment by neilbd** [comment]
Schedule module is almost done. I have a problem with spacing each piece of data, as you can see in this pic below

![image](https://cloud.githubusercontent.com/assets/8405298/17463592/05183342-5c97-11e6-9cb7-9814acf7970f.png)

My code for the table in Jade is 
`.shell.blue.schedule-shell
    #schedule.anchor
    .container
        .subsection.row.content
            .col-lg-8.col-lg-offset-2.col-md-10.col-md-offset-1.content
                h2 Schedule
                p Our schedule for HackNC! Click ____ for our complete list of workshops

```
            .col-md-2
                h3 Friday
                table#friday(style="width:100%")

            .col-sm-4.col-md-2.col-md-offset-2
                h3 Saturday
                table#saturday(style="width:100%")

            .col-sm-4.col-md-2.col-md-offset-2
                h3 Sunday
                table#sunday(style="width:100%")`
```

Have not uploaded these changes to GitHub

**Comment by neilbd** [comment]
Created new changes for the schedule in **neil_schedule** branch. Here is a pic of what it looks like now [
![screen shot 2016-08-08 at 9 54 32 pm](https://cloud.githubusercontent.com/assets/8405298/17502057/cf5f0146-5db2-11e6-8863-6940ce13fee8.png)
](url)

**Comment by subdavis** [comment]
Cool!  Since we don't _have_ a schedule, we don't want this on the homepage yet.  We'll keep it for when it's ready though!

**Comment by subdavis** [comment]
Why is there a copy of index.html and index.css back in the gulp project?  I think we refactored those.

**Comment by neilbd** [comment]
Yeah, was going to clean the schedule up when it's time to merge. I'm not sure why there is an index.html and index.css in Gulp, but it hasn't affected the code output. I'll clean it up though
